engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/details/views/content-host-details.html

```diff
@@ -123,6 +123,26 @@
           </li>
         </ul>
       </li>
+      <li uib-dropdown
+          ng-class="{active: stateIncludes('content-host.debs-actions') || stateIncludes('content-host.debs-installed') || stateIncludes('content-host.debs-upgradeable')}" >
```

Comment:
Is there a way to hide this depending on os_family?
`ng-hide="host.operatingsystem.family == 'Debian'"` did not work...

Reply:
Will need to look into how to have the OS created on registration. Not sure where we are doing that today. But if the OS is there I think that would work. Once that's figured out it might be cleaner to extend the hosts API to have a field which says "debian: true/false" based on this logic and we can use that in the view